_tutorials/101_quickstart/101_quickstart_step-01-desc.html

```diff
@@ -20,6 +20,6 @@
         PS: If you're not too familiar with XML: Whenever we refer to attributes, we prepend their name with the @-sign. 
         This is common practice and originates in the <a href="https://en.wikipedia.org/wiki/XPath" target="_blank" rel="noopener, noreferrer">XPath</a>
         language. When writing them in XML, you must not include the @-sign, but write the attribute's name only. Feel
-        free to have a look at our <a href="/tutorials/102-structure.html" arget="_blank" rel="noopener, noreferrer">tutorial on XML basics</a>.
+        free to have a look at our <a href="/tutorials/100-structure.html" arget="_blank" rel="noopener, noreferrer">tutorial on XML basics</a>.
```

Comment:
Could you fix the `arget="_blank"` to `target=...` , too, please?

Reply:
sorry, too fast ;-)

Reply:
fixed it manually in master

Reply:
Thx 👍